core/domain/exp_domain.py

```diff
@@ -758,17 +758,17 @@ def validate(self, strict=False):
         for state_name, state in self.states.iteritems():
             for param_change in state.param_changes:
                 param_change.validate()
+                if param_change.name in feconf.INVALID_PARAMETER_NAMES:
+                    raise utils.ValidationError(
```

Comment:
Thanks for adding this, this looks more like a state validation check, can you please see whether we have such check in stat.validate?

Also, my major concern is whether this can create an issue in the existing exploration (in production)?

Reply:
Ah I did not add this check I just shifted it before the `if param_change.name not in self.param_specs:` check. This is because if we want to check for INVALID_PARAMETER_NAMES, then it surely won't be in self.param_specs. Hence this check would be unreachable.